turbopack/crates/turbopack-ecmascript/src/references/mod.rs

```diff
@@ -3312,10 +3318,16 @@ async fn resolve_as_webpack_runtime(
     }
 }
 
-#[turbo_tasks::value(transparent)]
-#[derive(Hash, Debug, Clone)]
+#[derive(Hash, Debug, Clone, Eq, Serialize, Deserialize, PartialEq, TraceRawVcs)]
 pub struct AstPath(#[turbo_tasks(trace_ignore)] Vec<AstParentKind>);
 
+impl TaskInput for AstPath {
+    fn is_transient(&self) -> bool {
+        false
+    }
+}
+unsafe impl NonLocalValue for AstPath {}
```

Comment:
@bgw @sokra Not entirely sure if these manual `TaskInput` and `NonLocalValue` impls are correct? `AstParentKind` is a type from swc